robots.txt

```diff
@@ -1,4 +1,3 @@
 User-Agent: *
 Disallow: /
```

Comment:
Should be "Allow: /". Once this is synced with the prod, we need to have Disallow in the dev and make it permanent in dev